src/collision/cm_test.c

```diff
@@ -202,6 +202,9 @@ void Cm_InitBoxHull(void) {
  */
 int32_t Cm_SetBoxHull(const vec3_t mins, const vec3_t maxs, const int32_t contents) {
 
+	VectorCopy(mins, cm_box.brush->mins);
+	VectorCopy(maxs, cm_box.brush->maxs);
+	
```

Comment:
And here it is, ladies and gentlemen. The source of an entire weekend full of frustration. 2 lines of code. It's also worth noting that this bug is almost 2 years old.